spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+
+# Defines the test link string
+TEST_LINK = "ofn test link"
+
+# Defines the test href string. This one should contain all non-encoding characters, for a URL
+TEST_HREF = ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+describe 'add dynamic content to homepage' do
+	include AuthenticationHelper
+
+	before(:each) do
+		login_as_admin_and_visit spree.edit_admin_general_settings_path
+	end 
+	
+		
+	context 'edit the Content section', js: true do	
+		before(:each) do
+			click_link ("Content")
+		end 
+
+
+		# makes sure we are on the right page
+		scenario 'while on the Content section' do
+			
+			#Q1: option 1 and/or
+			expect(page).to have_current_path("/admin/contents/edit")
+
+			#option 2?
+			expect(page).to have_content("FOOTER AND EXTERNAL LINKS")
+
+		end
+		
+
+		scenario 'insert content on Footer and Links and check homepage' do
+			
```

Comment:
I think the assertion on the previous test should be put here as this test starts. it speeds up things.

Reply:
Whoa - it did. Reduced the test time in around 1 sec. Awesome!